doc/source/tutorials/local/Adding_data/Adding_observation_data.rst

```diff
@@ -212,6 +212,40 @@ species and instrument and do not need an accompanying precisions file. These ca
 The data will be processed in the same way as the old AGAGE data, and stored in the object store accordingly.
 Ensure that the ``source_format`` argument matches the input filetype, as the two are not compatible.
 
+2. Adding and standardising column data
+--------------------------------
```

Comment:
Similarly to above, the number of `-` characters should be increased to match the length of the new title.